tdrs-backend/tdpservice/users/serializers.py

```diff
@@ -203,6 +205,316 @@ def update(self, instance, validated_data):
         return instance
 
 
+class UserProfileChangeRequestSerializer(UserProfileSerializer):
+    """Serializer for user profile updates that creates change requests instead of direct updates."""
+
+    # Add a field to indicate whether to create change requests or update directly
+    create_change_requests = serializers.BooleanField(default=True, write_only=True)
```

Comment:
personally, i don't think this needs to be an option. you already have the permissions guard allowing only admins to make updates directly, i think that's sufficient - admins make direct updates and users request changes for admin approval.